src/NativeModules/UIManager.js

```diff
@@ -61,6 +61,11 @@ const UIManager = {
   configureNextLayoutAnimation(callback, errorCallback) {
 
   },
+  AndroidDrawerLayout: {
+    Constants: {
+      DrawerPosition: { Left: 8388611, Right: 8388613 },
```

Comment:
I know: https://developer.android.com/reference/android/view/Gravity.html#END and https://developer.android.com/reference/android/view/Gravity.html#START

That are the values used in the AndroidDrawerLayout: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/views/drawer/ReactDrawerLayoutManager.java#L145